openwisp_users/migrations/0003_default_organization.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.1.1 on 2018-09-16 12:20
+
+from django.db import migrations
+from django.conf import settings
+
+
+def set_default_organization_uuid(apps, schema_editor):
+    """
+    Get default organzation or create a default
+    organization and set settings.DEFAULT_ORG_UUID
+    """
+    organization = apps.get_model('openwisp_users', 'organization')
+    try:
```

Comment:
I think the first thing you have to do is to check whether any organization already exists, if it does just return and do nothing. This will avoid creating new default organizations in already existing openwisp instances.